.github/workflows/rgw-multisite-test/action.yml

```diff
@@ -76,11 +76,17 @@ runs:
     - name: RGW configuration period should be committed on first reconcile and not be committed on second reconcile
       shell: bash --noprofile --norc -eo pipefail -x {0}
       run: |
-        ns_name_primary='"rook-ceph/multisite-store"' # double quotes intended
-        ns_name_secondary='"rook-ceph-secondary/zone-b-multisite-store"' # double quotes intended
+        ns_name_primary='"rook-ceph/zone-a"' # double quotes intended
+        ns_name_secondary='"rook-ceph-secondary/zone-b"' # double quotes intended
         committed_msg="committing changes to RGW configuration period for CephObjectStore"
         tests/scripts/github-action-helper.sh verify_operator_log_message "${committed_msg} ${ns_name_primary}"
         tests/scripts/github-action-helper.sh verify_operator_log_message "${committed_msg} ${ns_name_secondary}"
+        # check for no period update, update the zone endpoint list as empty in the ceph object zone CR
```

Comment:
This test is still in progress?

Reply:
Now it is done, I was trying different combination and planning to have minimal changes to test